From ccc72c5391fb1676d3e386368ef927b2cc5056d2 Mon Sep 17 00:00:00 2001
From: Zhipeng Gong <zhipeng.gong@intel.com>
Date: Fri, 4 Mar 2016 10:54:00 +0800
Subject: [PATCH 102/153] [VPG]: drm/i915: fix KW issues

Conflicts:
	drivers/gpu/drm/i915/i915_gem_execbuffer.c
	drivers/gpu/drm/i915/intel_lrc.c
	drivers/gpu/drm/i915/intel_ringbuffer.c
---
 drivers/gpu/drm/i915/i915_gem_execbuffer.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_gem_execbuffer.c b/drivers/gpu/drm/i915/i915_gem_execbuffer.c
index eba5659..5dce979 100644
--- a/drivers/gpu/drm/i915/i915_gem_execbuffer.c
+++ b/drivers/gpu/drm/i915/i915_gem_execbuffer.c
@@ -1766,7 +1766,8 @@ i915_gem_do_execbuffer(struct drm_device *dev, void *data,
 		obj = to_intel_bo(drm_gem_object_lookup(dev, file,
 							  exec[i].handle));
 		qe.objs[i].obj       = obj;
-		qe.objs[i].read_only = obj->base.pending_write_domain == 0;
+		if (obj)
+			qe.objs[i].read_only = obj->base.pending_write_domain == 0;
 
 	}
 	qe.num_objs = i;
-- 
1.7.1

